pyfolio/plotting.py

```diff
@@ -573,6 +574,24 @@ def plot_rolling_returns(
         The axes that were plotted on.
 
 """
+    def draw_cone(returns, num_stdev, live_start_date, ax):
```

Comment:
Because I wanted to be consistent with the parameter name in timeseries.cone_rolling() which is the only place it gets passed. Is it worth changing for simplicity, or worth keeping because better to be consistent with where it is used downstream? thoughts?